### PR TITLE
update widgets and add new options

### PR DESCRIPTION
**Pull request by gebr11**
modify UI widgets, add new properties and fix widths

### CODE REVIEW

File: SLC-AS-EVSNeuronScript_1/PopupDialog.cs [hunk 1]
```diff
@@ -86,40 +167,60 @@ internal void SetDefaultData(IEngine engine, string elementData)
...
 
         internal void UpdateDialogData()
         {
-            var newDropdownValue = this.VideoPathDropDown.Selected;
             var tableData = dmsElement.GetTable(VideoPathTableId).GetData();
-            var matchedRow = tableData[newDropdownValue];
-            this.FrameDelayTextBox.Text = Convert.ToString(matchedRow[6]);
-            this.VerticalDelayTextBox.Text = Convert.ToString(matchedRow[7]);
-            this.HorizontalDelayTextBox.Text = Convert.ToString(matchedRow[8]);
+            var matchedRow = tableData[VideoPathDropDown.Selected];
```
Comment (BlakeSLC) [inline SLC-AS-EVSNeuronScript_1/PopupDialog.cs]: either we don't need to use tabledata again here, just use it when previously requested (line 165) OR add a small sanity check if the row given the videopathdropdown.selected no longer exists

**Review by BlakeSLC — APPROVED** [review]
minor remark

File: SLC-AS-EVSNeuronScript_1/PopupDialog.cs (hunk 1 above)
Comment (gebr11) [reply SLC-AS-EVSNeuronScript_1/PopupDialog.cs]: fixed

**Comment by TomW-Skyline** [comment]
@gebr11 @BlakeSLC on request of Joey, I've added a button to reset to all values to the default, and a close button to avoid the abort script message.
image

**Pull request merged**